pyvrp/PenaltyManager.py

```diff
@@ -119,6 +121,10 @@ class PenaltyManager:
         PenaltyManager parameters. If not provided, a default will be used.
     """
 
+    MIN_PENALTY = 1
+    MAX_PENALTY = 100_000
+    FEAS_TOL = 0.05
```

Comment:
These should not be hardcoded inside the `_compute` method. This is still not great, but at least a little bit better.